docs/content/reference/ransomware.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Ransomware"
+date: 2021-06-23T18:13:59+05:30
+draft: true
+pre: '<i class="fas fa-lock"></i> '
+weight: 10
+---
+
+The Infection Monkey has the capability of simulating a ransomware attack on your network.
+All actions performed by the encryption routine are designed to be safe for production
+environments.
+
+To ensure minimum interference and easy recoverability, the ransomware simulation will encrypt
+files if the user specifies a directory that contains files that are safe to encrypt.
+If no directory is specified, no files will be encrypted.
+
+<!-- add config screenshot here -->
+
+## How are the files encrypted?
+
+Files are "encrypted" in place with a simple bit flip. Encrypted files are renamed to have
+`.m0nk3y` appended to their names.
+
+This is a safe way to simulate encryption since it is easy to "decrypt" your files. You can simply perform a bit flip on the files again and rename them to remove the appended `.m0nk3y` extension.
+
+This is sufficient for a ransomware simulation as your files are unusuable and are renamed with a different extension, similar to how many ransomwares act. These changes should trigger your security solutions.
+
+
+## Which files are encrypted?
+
+All regular files with [relevant extensions](#relevant-file-extensions-for-encryption) in the
+configured directory are attempted to be encrypted during the simulation.
+
+The simulation is not recursive, i.e. it will not touch any files in sub-directories of the
+configured directory — only appropriate files in the top level of the tree.
```

Comment:
"top level of the tree" implies `C:\` or `/` to me, which would scare users and isn't what we mean here.